src/main/java/org/zalando/nakadi/exceptions/runtime/AuthorizationSectionException.java

```diff
@@ -0,0 +1,8 @@
+package org.zalando.nakadi.exceptions.runtime;
+
+public class AuthorizationSectionException extends NakadiBaseException {
```

Comment:
Maybe the name of the exception could be more clear like `AuthorizationSectionAbsentException` or `AuthorizationAbsentException`? What do you think?